signup.php

```diff
@@ -105,7 +105,7 @@ function checkAvailability() {
                 <div class="col-md-9 col-md-offset-1">
                     <div class="panel panel-danger">
                         <div class="panel-heading">
-                            SINGUP FORM
+                            SIGN UP FORM
```

Comment:
when refactoring the file, please include removing lines without code (look at line 116 & 117 for reference)